src/components/LogoGrid/LogoGrid.tsx

```diff
@@ -31,6 +31,13 @@ export const PartnerLogoGrid = () => {
       >
         <Logo src="/images/grand-rounds.png" />
       </a>
+      <a
+        href="https://globalhealth.harvard.edu/"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        <Logo src="/images/harvard.png" />
+      </a>
```

Comment:
Consider using the `ExternalLink` component and add the `alt` attribute for the logo (see https://github.com/covid-projections/covid-projections/pull/1426/files#diff-7d532764b5cc24c40e25317a32e748a5R14)